test/common/services/deployment.ts

```diff
@@ -35,17 +35,7 @@ export function DeploymentProvider({ getService }: FtrProviderContext) {
      * Useful for functional testing in cloud environment
      */
     async isOss() {
-      const baseUrl = this.getEsHostPort();
-      const username = config.get('servers.elasticsearch.username');
-      const password = config.get('servers.elasticsearch.password');
-      const response = await fetch(baseUrl + '/_xpack', {
-        method: 'get',
-        headers: {
-          'Content-Type': 'application/json',
-          Authorization: 'Basic ' + Buffer.from(username + ':' + password).toString('base64'),
-        },
-      });
-      return response.status !== 200;
+      return config.get('kbnTestServer.serverArgs').indexOf('--oss') > -1;
```

Comment:
Pretty sure this function used to work when run against remote instances. Is there a reason for removing that capability?

Reply:
The method was checking to see if Elasticsearch was oss, and it's now checking the Kibana configuration. The few instances where checking ES was required have been updated to always assume basic+. I see this is a stop gap to satisfy the current tests, as we will no longer be producing oss builds and this code will be removed eventually.